nlp/Dockerfile

```diff
@@ -4,13 +4,48 @@ FROM birdhouse/pavics-jupyter-base:0.2.1
 USER root
 
 COPY environment.yml /environment.yml
+COPY /nl2query ./nl2query 
+COPY notebook_config.yml /notebook_config.yml
+
 
 # update env "birdy"
 # use umask 0000 so that package files for the updated environment are usable by the user for the jupyter-conda-extension
 RUN umask 0000 && conda env update -f /environment.yml
 
-COPY notebook_config.yml /notebook_config.yml
 
-# specify user because of problem running start-notebook.sh when being root
-USER jenkins
 
+# Installing java opjdk
+RUN conda install -c anaconda openjdk
+ENV LANG=C.UTF-8
+
+
+# Download spacy model
+RUN python -m spacy download en_core_web_trf
+
+# Heideltime Tree-tagger Installation 
+RUN cd nl2query/heideltime && \
+    mkdir tree-tagger-linux-3.2.3 && \
+    cd tree-tagger-linux-3.2.3 && \
+    curl -o tree-tagger-linux-3.2.3.tar.gz  https://www.cis.lmu.de/~schmid/tools/TreeTagger/data/tree-tagger-linux-3.2.3.tar.gz && \
+    curl -o tagger-scripts.tar.gz https://www.cis.lmu.de/~schmid/tools/TreeTagger/data/tagger-scripts.tar.gz && \
+    curl -o english.par.gz https://www.cis.lmu.de/~schmid/tools/TreeTagger/data/english.par.gz && \
```

Comment:
are these two files used, before deletion?

Reply:
Yes they are used, they are part of the installation steps from https://www.cis.lmu.de/~schmid/tools/TreeTagger/

Reply:
But they are deleted right after download, at line 35..
It's not dramatic, but I'm wondering why these files are getting downloaded

Reply:
Yes excactly,  `english.par.gz` and `tagger-scripts.tar.gz` are packages that are consumed by `install-tagger.sh`. So they are 
essential for the installation of tree-tagger and for accepting english language.